pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "foursight-cgap"
-version = "0.3.4"
+version = "0.3.5"
```

Comment:
since this is a new check you can increment the minor version instead (semver is major.minor.patch)